### PR TITLE
Display link to repository after branch deletion

### LINKED ISSUE #1487: Keep link to repository for deleted branch
#1 discussed about link to branches. I delete all my merged branches (because they are available in PR anyway), and the branch link is no longer available, which makes sense. However I would like the “user” part still points to the repository. For example, in #1474 I want `stipsan` to link to <https://github.com/stipsan/refined-github>.

### PR DESCRIPTION
**Pull request by rzpio**
Closes #1487.

The link is still crossed out and "Deleted" is still displayed, but it's clickable and you will be redirected to the repository.

### CODE REVIEW

File: source/features/linkify-branch-refs.js [hunk 1]
```diff
@@ -17,19 +17,27 @@ function inPR() {
...
 			el.textContent.trim(),
-			`/${repo}/tree/${encodeURIComponent(branch)}`
+			`/${repo}`
 		);
+		if (branchName !== deletedBranch) {
+			urls.set(
+				el.textContent.trim(),
+				`/${repo}/tree/${encodeURIComponent(branch)}`
+			);
+		}
```
Comment (bfred-it) [inline source/features/linkify-branch-refs.js]: Why is this change needed? Isn’t the url the same whether it’s deleted or not?

**Comment by sindresorhus** [comment]
@rzpio Do you have a test PR I can try this out on?
